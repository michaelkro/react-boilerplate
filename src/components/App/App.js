import React, { Component } from 'react';
import { hot } from 'react-hot-loader/root';

class App extends Component {
  render() {
    return <div>Hello there</div>;
  }
}

export default hot(App);
