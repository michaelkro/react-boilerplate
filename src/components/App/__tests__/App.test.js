import React from 'react';
import { shallow } from 'enzyme';

import App from '../App';

test('it renders properly', () => {
  const wrapper = shallow(<App />);

  expect(wrapper).toMatchSnapshot();
});
