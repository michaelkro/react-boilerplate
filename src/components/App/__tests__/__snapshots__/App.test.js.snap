// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`it renders properly 1`] = `
<div>
  Hello there
</div>
`;
